### ISSUE TITLE
No return when making API call on Android with publish permissions.

### ISSUE THREAD

**Issue by poocart** [Wizcorp/phonegap-facebook-plugin#1171]
Cannot get any response from Java class. The issue where the crash begins is when checking if the active session contains permission being sent. First time permission approval is shown and permission is gained, then no return to application. Next time when application is restarted and the current session permissions are being checked there is no permission which was granted before.

**Comment by onlyfrank** [comment]
same problem here :( did you find a solution? 

second call to facebookConnectPlugin.api( "me/?fields=id,name,picture.type(large)... will NOT work if users have previously accepted permissions.

Any news?

**Comment by poocart** [comment]
@onlyfrank no, still waiting for plugin update on this :( how about you? have you found where is the bug?

**Comment by dinasalah** [comment]
Same happen with me, did any find a solution.
This is my code:
var url= fb_event_id+"/attending?method=POST&access_token="+fb_access_token;

facebookConnectPlugin.api(url, ["rsvp_event"], function (result) {
        alert("Result: " + JSON.stringify(result));
    },
    function (error) {
        alert("Failed: " + error);
    });
it opens facebook to accept the new permission "rsvp_event" , then I press "Ok" to accept permission , and it returns to my app with this result in logcat:
    02-03 09:24:54.410: D/CordovaActivity(13432): Incoming Result. Request code = 64206
    02-03 09:24:54.410: D/CordovaActivity(13432): We have a callback to send this result to
    02-03 09:24:54.410: D/ConnectPlugin(13432): activity result in plugin: requestCode(64206), resultCode(-1)

and neither the sucess function nor the failure function be called.

**Comment by rastafan** [comment]
Bump, this is happening to me too, rsvp_event as permission.
I tried with Test users, since i have not submitted this app to FB Review yet. It works fine when using an user that had the "Authorize Test Users for this app" checked in the Test User creation screen. It does not work when you give permission from the app itself.

Here is my code:

```
`facebookConnectPlugin.api("/[EVENT_ID]/attending", ["rsvp_event"], function (response) {
    //SUCCESS
    alert("OK: " + JSON.stringify(response));

}, function (error) {
    //FAIL
    alert("FAIL: " + JSON.stringify(error));
});`
```

Neither Success nor fail callback are called after the user gives permission. Also, you get a "an attempt was made to request new permissions for a session that has a pending request." after the first request attempt.
